src/services/user.js

```diff
@@ -7,10 +7,18 @@ import http from './http';
 import session from './session';
 import storage from './storage';
 
+/**
+ * @global
+ * @typedef User
+ * @property {int} current user's current cl&w credits
+ * @property {int} required user's required cl&w credits
+ */
+
 /**
  * Get events attended by the user
- * @param {String} sessionCode Identifier for a session
- * @return {Promise.<attendedEvents[]>}
+ * @param {String} termCode code for the semester
+ * @param {String} username username of the user
```

Comment:
Swap this line with the previous line. `@param` definitions should be in the order of the actual functions parameters.